### ISSUE TITLE
TeachingTip: Regression - Background of custom content of light-dismissable tip no longer matches rest of the background

### ISSUE THREAD

**Issue by Felix-Dev** [microsoft/microsoft-ui-xaml#3436]
**Describe the bug**
The newest WinUI 2.5 prerelease comes with a TeachingTip regression (presumably caused by PR https://github.com/microsoft/microsoft-ui-xaml/pull/3205). The background of the custom content of a *light-dismissable* tip no longer uses the background used for the rest of the tip's UI.

```xml
<muxc:TeachingTip Title="Title" Subtitle="Subtitle"
              IsLightDismissEnabled="True">
    <TextBlock Text="Extra Content" Margin="0,10,0,0" />
</muxc:TeachingTip>
```
![image](https://user-images.githubusercontent.com/1398851/96257385-8f73d980-0fba-11eb-8a44-8798308d9cbb.png)

**Expected behavior**
![image](https://user-images.githubusercontent.com/1398851/96257322-72d7a180-0fba-11eb-8aa0-a88c4d5320fd.png)

**Version Info**

NuGet package version: 
Microsoft.UI.Xaml v2.5.0-prerelease.200923002

**Additional context**
A PR is in the works.

**Comment by StephenLPeters** [comment]
eeek, thanks we should fix this before the 2.5 full release.

**Comment by Felix-Dev** [comment]
The hero content background is also not following the rest of the background of a light-dismissable tip:
```xml
<muxc:TeachingTip Title="Title" Subtitle="Subtitle"
              IsOpen="True"
              IsLightDismissEnabled="True">
    <muxc:TeachingTip.HeroContent>
        <Rectangle Fill="Transparent" Width="100" Height="70" />
    </muxc:TeachingTip.HeroContent>
</muxc:TeachingTip>
```
![image](https://user-images.githubusercontent.com/1398851/96336072-62d4c600-107d-11eb-97cc-830a17ce64ec.png)

This happens since the introduction of the TeachingTip control in WinUI 2.1. I think this is a bug too as I would expect the HeroContent's default background to match that of the rest of the tip.
